Add possibility to download latest build (#2378)
kyma-project/cli: README.md

## Install

> For the installation instructions follow steps in [this article](./docs/user/README.md#how-to-install).

Reviewer: ```suggestion
For the installation instructions, see [How to Install](./docs/user/README.md#how-to-install).
```
